developer/src/tike/dialogs/UfrmNew.pas

```diff
@@ -321,6 +325,14 @@ procedure TfrmNew.mnuShowIconsClick(Sender: TObject);
   lvItems.ViewStyle := vsIcon;
 end;
 
+procedure TfrmNew.SetCanAddToProject(const Value: Boolean);
+begin
+  FCanAddToProject := Value;
+  if not Value then
+    chkAddToProject.Checked := False;
```

Comment:
Does `chkAddToProject.Checked` default to True?

Reply:
Yes:

https://github.com/keymanapp/keyman/blob/070ead74cf88e96b932184a07d57a692a9a86077/developer/src/tike/dialogs/UfrmNew.dfm#L100